### DISCUSSION TITLE
Wait between repositories?

### DISCUSSION THREAD

**Discussion by FalconerTC** [renovatebot/renovate#11628] [Request Help]
I run slef-hosted Renovate with autoDiscover and it's going repo to repo too quickly. I haven't hit Github API rate limiting but I run Renovate on GKE with GCP NAT Gateway and I am seeing dropped packets because of too rapid subsequent requests to the same endpoint. Any way to add a delay between each repo hit by Renovate?

**Comment by viceice** [comment]
Nope, not supported.
